README.md

```diff
@@ -2,6 +2,14 @@
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/00cf8469d692073171ce/maintainability)](https://codeclimate.com/github/ministryofjustice/offender-management-allocation-api/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/00cf8469d692073171ce/test_coverage)](https://codeclimate.com/github/ministryofjustice/offender-management-allocation-api/test_coverage) [![CircleCI](https://circleci.com/gh/ministryofjustice/offender-management-allocation-api.svg?style=svg)](https://circleci.com/gh/ministryofjustice/offender-management-allocation-api)
 
+## Setup
+
+Install the git pre-commit hook before you start working on this repository.
+From the root of the repo:
+
+```
+ln -s ../../config/git-hooks/pre-commit.sh .git/hooks/pre-commit
```

Comment:
Is this path correct - from the root of the repo shouldn't it be `./config/git-hooks ... `

Reply:
👍